### ISSUE TITLE
Use Node 6

### ISSUE THREAD

**Issue by macdonst** [phonegap/phonegap-template-vue-f7-blank#1]
Let's just force Node 6 so we can use ES2015 style JS.

**Comment by devgeeks** [comment]
Yeah, I was tempted. It's really just in the configs etc where it affects things, so I was willing to backpedal that a bit.

6.x is current LTS, but there are probably a _lot_ of users still on older versions...

**Comment by devgeeks** [comment]
I think it's time to make a call on this and require 6.

**Comment by MarcGodard** [comment]
I agree.
